tasks/containers/TaskWorker.js

```diff
@@ -79,14 +79,32 @@ export default compose(
 
```

Comment:
I'm getting an infinite `taskWork` request here because this is always returning true and I'm  struggling to find the reason why.

@ahdinosaur @iainkirkpatrick If either of you two gets a chance to look at this that'd be rad as I'm feeling blocked. Otherwise, coming back to it with fresh eyes tomorrow might be enough.

This branch also depends on this file to be changed https://github.com/ahdinosaur/feathers-action/pull/20/files to work